nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentsMigrator.java

```diff
@@ -139,6 +256,36 @@ protected void migrateComments(CoreSession session, RelationManager relationMana
 
         Graph graph = relationManager.getGraph(config.graphName, session);
         graph.remove(statement);
+
+    }
+
+    /**
+     * @since 11.1
+     */
+    protected void stripACLs(DocumentModel comment) {
+        ACP acp = comment.getACP();
+        acp.removeACL(LOCAL_ACL);
+        comment.setACP(acp, true);
+    }
+
+    /**
+     * @since 11.1
+     */
+    protected void migrateCommentsFromPropertyToSecured(CoreSession session, PropertyCommentManager commentManager,
+            DocumentModel comment) {
+        comment.putContextData(DISABLE_NOTIFICATION_SERVICE, TRUE); // Remove notifications
```

Comment:
Not sure it's needed as after this line we're using the reference, the id or ACL.

Reply:
In `org.nuxeo.ecm.core.api.AbstractSession#move(org.nuxeo.ecm.core.api.DocumentRef, org.nuxeo.ecm.core.api.DocumentRef, java.lang.String)` will dispatch different events : `ABOUT_TO_MOVE` and `DOCUMENT_MOVED`, as it's a job migration we disable notification as its done on others cases.